polytope/polytope.py

```diff
@@ -1292,25 +1301,26 @@ def volume(polyreg):
         N = 50
     elif n == 2:
         N = 500
-    elif n ==3:
+    elif n == 3:
         N = 3000
     else:
         N = 10000
 
     l_b, u_b = polyreg.bounding_box
-    x = np.tile(l_b,(1,N)) +\
-        np.random.rand(n,N) *\
-        np.tile(u_b-l_b,(1,N) )
+    x = np.tile(l_b, (1, N)) +\
+        np.random.rand(n, N) *\
+        np.tile(u_b-l_b, (1, N))
     aux = np.dot(polyreg.A, x) -\
```

Comment:
Insofar as we are eliminating line continuation via `\`, how about changing this to the following?
```python
aux = (np.dot(polyreg.A, x)
           - np.tile(
               np.array([polyreg.b]).T,
               (1, N)
           ))
```
Your original changes did not modify this statement, so OK to leave it as it already is.